src/w3c/templates/headers.js

```diff
@@ -11,13 +11,15 @@ function getSpecTitleElem (conf) {
     specTitleElem.remove();
     conf.title = specTitleElem.textContent.trim();
   } else {
-    if (document.title !== conf.title) {
-      pub("warn", "The document's title and the `<title>` element differ.");
-    }
     specTitleElem.textContent = conf.title;
     specTitleElem.id = "title";
   }
   specTitleElem.classList.add("title", "p-name");
+  if (document.querySelector("title") === null) {
+    document.title = conf.title;
```

Comment:
Will `document.title` remain alive after respec2html?

Reply:
Yep, it literally adds a new title html element to the doc.

Reply:
I'll add a test for this.